### PR TITLE
Add test link for the Shortcuts app

### PR DESCRIPTION
**Pull request by miasma13**
Test link that on iOS launches the Shortcuts app and runs the shortcut with a given name.

### CODE REVIEW

File: features/url-schemes.html [hunk 1]
```diff
@@ -16,6 +16,7 @@ <h1>URL Schemes</h1>
  <li><a href="custom://scheme/with.path">custom://scheme/with.path</a> - a custom URL scheme that probably isn't supported</li>
  <li><a href="intent://duckduckgo.com/?q=test#Intent;scheme=https;package=com.duckduckgo.mobile.android;S.browser_fallback_url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.duckduckgo.mobile.android;end">intent://duckduck.com</a> - launches DuckDuckGo Android app using a query parameter or opens the PlayStore</li>
  <li><a href="intent://duckduckgo.com#Intent;scheme=https;package=com.duckduckgo.mobile.android;S.browser_fallback_url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.duckduckgo.mobile.android;end">intent://duckduck.com</a> - launches DuckDuckGo Android app or opens the PlayStore</li>
+ <li><a href="shortcuts://run-shortcut?name="Test%20shortcut">shortcuts://run-shortcut?name="Test%20shortcut</a> - on iOS opens Shortcuts app and triggers execution of "Test shortcut" shortcut</li>
```
Comment (kdzwinel) [inline features/url-schemes.html:19]:
```suggestion
 <li><a href="shortcuts://run-shortcut?name="Test%20shortcut">shortcuts://run-shortcut?name="Test%20shortcut"</a> - on iOS opens Shortcuts app and triggers execution of "Test shortcut" shortcut</li>
```

**Review by kdzwinel — APPROVED** [review]
I found a small nit, feel free to merge after this. Thanks!

File: features/url-schemes.html (hunk 1 above)
Comment (miasma13) [reply features/url-schemes.html:19]: Thanks for looking at it. Actually the quotes around the name value are not required and there was a stray quote inside href. I will fix it in the following commit.

**Review by kdzwinel — APPROVED** [review]
Ah, I see, glad you cached it. Thanks Michał!

**Pull request merged**